code/modules/power/supermatter/supermatter.dm

```diff
@@ -923,7 +923,38 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	. = ..()
 	if(.)
 		return
-	dust_mob(user, cause = "hand")
+	if(user.incorporeal_move || user.status_flags & GODMODE)
+		return
+
+	. = TRUE
+	if(user.combat_mode || (user.zone_selected != BODY_ZONE_PRECISE_MOUTH))
```

Comment:
does this /really/ need to check combat mode?

Reply:
It's probably not strictly necessary, but I might as well.

Reply:
Let's not, please

Reply:
No, we should be limiting the amount of combat mode checks.

Reply:
(and be removing any that are left so we don't require modal interactions)